CONTRIBUTING.md

```diff
@@ -280,27 +280,13 @@ C:\..\> set LOG_LEVEL="INFO"
 C:\..\> python -m unittest test/python/circuit/test_circuit_operations.py
 ```
 
-##### Online Tests
-
-Some tests require that you an IBMQ account configured. By default these
-tests are always skipped. If you want to run these tests locally please
-go to this
-[page](https://quantumexperience.ng.bluemix.net/qx/account/advanced) and
-register an account. Then you can either set the credentials explicitly
-with the `IBMQ_TOKEN` and `IBMQ_URL` environment variables to specify
-the token and url respectively for the IBMQ service. Alternatively, if
-you already have a single set of credentials configured in your
-environment (using a `.qiskitrc`) then you can just set
-`QISKIT_TESTS_USE_CREDENTIALS_FILE` to `1` and it will use that.
-
 ##### Test Skip Options
 
 How and which tests are executed is controlled by an environment
 variable, `QISKIT_TESTS`:
 
 Option | Description | Default
 ------ | ----------- | -------
-`skip_online` | Skips tests that require remote requests. Does not require user credentials. | `False`
```

Comment:
Maybe we should also remove `skip_online` as an option from `qiskit/test/testing_options.py`.

Reply:
I didn't want to remove that or the decorator because other things (mainly the ibmq provider) for better or worse use things from `qiskit.test` directly: https://github.com/Qiskit/qiskit-ibmq-provider/blob/22e9b8d0930f1af0a7305229c7379909b3459cd3/test/decorators.py#L34

Ideally we should remove this dependency and not treat terra's test helpers (especially a terrible one that doesn't use env vars correctly) as an external api. But that feels like something outside the scope of this pr.